lib/active_storage_support/base64_attach.rb

```diff
@@ -5,13 +5,18 @@ module Base64Attach
 
     def attachment_from_data(attachment)
       if attachment.is_a?(Hash)
-        base64_data = attachment.delete(:data)
-        if base64_data.try(:is_a?, String) && base64_data =~ /^data:(.*?);(.*?),(.*)$/
+        attachment.symbolize_keys!
```

Comment:
I think I prefer using `.with_indifferent_access` to avoid changing the original hash. Thoughts?

https://apidock.com/rails/Hash/with_indifferent_access

Reply:
Don't know if using this will solve the argument error issue, but let me give it a try.

Reply:
Yeah, it definitely does not solve the argument error issue from activestorage, so I believe it should stay this way.